src/PKSim.Core/Snapshots/Mappers/CompoundPropertiesMapper.cs

```diff
@@ -73,6 +84,29 @@ private CompoundProcessSelection compoundProcessSelectionFor(IReactionMapping re
          return compoundProcessSelection;
       }
 
+      private IReactionMapping reactionMappingFrom(CompoundProcessSelection compoundProcessSelection, Model.CompoundProcess process)
+      {
+         switch (process)
```

Comment:
Really only trying to save the very minimal info required to build the simulation from scratch